### PR TITLE
Do not open the sidebar automatically on small widths

### PR DESCRIPTION
**Pull request by szaimen**
<details>
<summary>For my own testing</summary>

```
docker run -it \
--name nextcloud-easy-test \
-p 8443:443 \
-e TRUSTED_DOMAIN=192.168.24.128 \
--volume="nextcloud_easy_test_npm_cache_volume:/var/www/.npm" \
-e SERVER_BRANCH=enh/noid/dont-automatically-open-sidebar-on-mobile \
-e VIEWER_BRANCH=master \
ghcr.io/szaimen/nextcloud-easy-test:latest
```

</details>

### CODE REVIEW

File: apps/files/js/filelist.js [hunk 1]
```diff
@@ -433,7 +433,9 @@
 							OCA.Files.Files.handleDownload(url);
 						}
 
-						OCA.Files.Sidebar.open(fileInfo.path);
+						if (document.documentElement.clientWidth < 1024 / 2) {
```
Comment (adityaraute) [inline apps/files/js/filelist.js:436]: What is the purpose of 1024 / 2? Why not put a value directly

**Review by adityaraute — CHANGES_REQUESTED** [review]

File: apps/files/js/filelist.js (hunk 1 above)
Comment (szaimen) [reply apps/files/js/filelist.js]: fixed

**Review by Pytal — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]

**Comment by szaimen** [comment]
CI failure inrelated

**Comment by ChristophWurst** [comment]
testing https://github.com/nextcloud/backportbot/pull/364

**Comment by szaimen** [comment]
Already manually backported. But we can close it again...

**Comment by ChristophWurst** [comment]
> testing [nextcloud/backportbot#364](https://github.com/nextcloud/backportbot/pull/364)

I know.

**Comment by ChristophWurst** [comment]
Yay. It fails :tada:

**Pull request merged**